stream/distributedlog/core/src/main/java/org/apache/distributedlog/AppendOnlyStreamWriter.java

```diff
@@ -68,6 +68,20 @@ public void force(boolean metadata) throws IOException {
         }
     }
 
+    public void flush(boolean waitForCompletion) throws IOException {
+      try {
+        if (waitForCompletion) {
+          FutureUtils.result(logWriter.flush());
+        } else {
+          logWriter.flush();
+        }
+      } catch (IOException ioe) {
+        throw ioe;
+      } catch (Exception ex) {
+        throw new UnexpectedException("unexpected exception in AppendOnlyStreamWriter.force", ex);
```

Comment:
Your comment is not changed... still **AppendOnlyStreamWriter.force**

Reply:
Fixed.